File: Java-Selenium/src/test/java/ru/stqa/training/selenium/LiteCart/LoginLocal.java
package ru.stqa.training.selenium.LiteCart;

import org.junit.Test;
import org.openqa.selenium.By;

import static org.openqa.selenium.support.ui.ExpectedConditions.titleIs;

public class LoginLocal extends TestBase {


    public void LoginAdmin(LoginData loginData){
        wd.get("http://localhost/litecart/admin/login.php");
        wd.findElement(By.name("username")).clear();
        wd.findElement(By.name("username")).sendKeys(loginData.getName());
        wd.findElement(By.name("password")).sendKeys(loginData.getPass());
        wd.findElement(By.name("login")).click();
        wait.until(titleIs("My Store"));
    }

}

File: NodeJS-Selenium/LiteCart/LoginLocal.js

const {Builder, By, until} = require('selenium-webdriver');
const test = require("selenium-webdriver/testing");

test.describe('Local Login', function() {

  let driver;

  test.before(function *() {
    driver = new Builder().forBrowser('chrome').build();
  });

   test.it('Login Local', function*() {
    driver.get('http://localhost/litecart/admin/login.php');
    driver.findElement(By.name('username')).sendKeys('admin');
    driver.findElement(By.name('password')).sendKeys('admin');
    driver.findElement(By.name('login')).click();
    driver.wait(until.titleIs('My Store'), 1000);
  });

  test.after(function(){
      driver.quit();
      }
    );
});
